.github/workflows/conformance-aks.yaml

```diff
@@ -146,9 +146,26 @@ jobs:
           echo "matrix=$(jq -c . < /tmp/result.json)" >> $GITHUB_OUTPUT
           echo "empty=$(jq '(.include | length) == 0' /tmp/result.json)" >> $GITHUB_OUTPUT
 
+  wait-for-images:
+    name: Wait for images
+    runs-on: ubuntu-24.04
+    timeout-minutes: 30
+    steps:
+      - name: Checkout context ref (trusted)
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          ref: ${{ inputs.context-ref || github.sha }}
+          persist-credentials: false
+
+      - name: Wait for images
+        uses: ./.github/actions/wait-for-images
+        with:
+          SHA: ${{ inputs.SHA || github.sha }}
+          images: cilium-ci operator-azure-ci hubble-relay-ci cilium-cli-ci
```

Comment:
Do we need `cilium-cli-ci` here?

Reply:
yea, so on main, action "install Cilium cli" fetches that image so we want to wait for it to be built. This is also default here:
https://github.com/cilium/cilium/blob/main/.github/actions/wait-for-images/action.yaml
For stable branches I can see that Joe added skipping waiting on clium-cli-ci image.